routes/web.php

```diff
@@ -108,6 +108,12 @@
         'as'=>'customers.'
     ], function(){ 
         Route::get('/customer', [CustomerController::class, 'index']);
+        Route::post('/customer', [CustomerController::class, 'store']); 
+        Route::get('edit-customer/{id}', [CustomerController::class, 'edit']);
+        Route::put('update-customer/{id}', [CustomerController::class, 'update']);
+        Route::delete('delete-customer/{id}', [CustomerController::class, 'destroy']);
+        
+        
```

Comment:
You can use instead
`/customer/{id}` for `edit`
`/customer/{id}` for `update` (they have different http request method so it should work fine)
`/customer/{id}` for `destroy` (different request method for delete)